packages/wonder-blocks-dropdown/src/components/__tests__/dropdown-opener.test.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from "react";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import DropdownOpener from "../dropdown-opener";
+
+describe("DropdownOpener", () => {
```

Comment:
I'm surprised there weren't already tests for DropdownOpener! Thanks for adding them!